### DISCUSSION TITLE
[Wasm] Windows.Storage, how to persist roaming folder ?

### DISCUSSION THREAD

**Discussion by giraudremy** [unoplatform/uno#8143] [Q&A] [answered]
Hi,

I follow this documentation to create file in Wasm application :
https://platform.uno/docs/articles/features/windows-storage.html

File is well created and I can read it until I close browser. When I restart browser, file is not existing...

Does roaming folder is persisted ?

I have made a sample :
[UnoApp1.zip](https://github.com/unoplatform/uno/files/8103954/UnoApp1.zip)

Write button will check if file exist and create it if not exist.
Read button will read the file in roaming folder.

if you restart browser and click to Read button, I have a FileNotFoundException
System.IO.FileNotFoundException: Could not find file '/roaming/Configuration/accounts.xml'.
File name: '/roaming/Configuration/accounts.xml' 

Why ?

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the repro! You've found a filesystem initialization issue, let's track it here: https://github.com/unoplatform/uno/issues/8157

**Reply by giraudremy** [reply]
Thank for you answer 👍
